docs/integrations/asgi.md

```diff
@@ -44,6 +44,11 @@ if __name__ == "__main__":
 
 You can read more about the OpenTelemetry ASGI middleware [here][opentelemetry-asgi].
 
+## Excluding URLs from instrumentation
+<!-- note that this section is duplicated for different frameworks but with slightly different links -->
+
+- [Quick guide](use-cases/web-frameworks.md#excluding-urls-from-instrumentation)
```

Comment:
Please add a note that `OpenTelemetryMiddleware` does accept an `excluded_urls` parameter (it's barely visible in the OTEL docs) but not an environment variable.

An `instrument_asgi` method would make this a bit more sensible, @Kludex what's happening with 
 https://github.com/pydantic/logfire/pull/324?